storage/ndb/src/kernel/blocks/dblqh/DblqhMain.cpp

```diff
@@ -17653,10 +17707,20 @@ void Dblqh::scanReleaseLocksLab(Signal* signal,
   check_send_scan_hb_rep(signal, scanPtr, regTcPtr);
   while (true)
   {
-    const Uint32 sig1 = 
-      get_acc_ptr_from_scan_record(scanPtr, 
-                                   scanPtr->scanReleaseCounter-1,
-                                   false);
+    Uint32 tmp = 0;
+    // Moz
+    if (scanPtr->m_aggregation) {
```

Comment:
Turn the if-statement around since no aggregation is the common case still

Reply:
OK